DigitalOffice/Services/ProjectService/Controllers/ProjectController.cs

```diff
@@ -1,16 +1,19 @@
-﻿using LT.DigitalOffice.ProjectService.Commands.Interfaces;
-using LT.DigitalOffice.ProjectService.Models;
+﻿using System;
 using Microsoft.AspNetCore.Mvc;
-using System;
+using LT.DigitalOffice.ProjectService.Models;
+using LT.DigitalOffice.ProjectService.Commands.Interfaces;
 
 namespace LT.DigitalOffice.ProjectService.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class ProjectController : ControllerBase
     {
-        [HttpGet("getProjectInfoById")]
-        public Project GetProjectInfoById([FromServices] IGetProjectInfoByIdCommand command, [FromQuery] Guid projectId)
+        [Route("getProjectInfoById")]
+        [HttpGet]
```

Comment:
Can be combined.

Reply:
there you go this is combined even though it's not my task or code